src/components/form-errors.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
I'm trying to set a phone number as one of the list
![image](https://user-images.githubusercontent.com/12034551/95961464-bfca4500-0e0d-11eb-8e84-0c5453bdcd7c.png)
Let's change our phone's RegExp little bit so that all our users will be valid

Reply:
can we give an embedding link to a existing user?
![image](https://user-images.githubusercontent.com/12034551/95961766-2bacad80-0e0e-11eb-8117-b90d77efb83a.png)

Reply:
Let's make our disabled Submit button some visual difference with an enabled one

Reply:
The required fields should be marked (e.g. '*')
Please see my original message about this task in private message in Teams to get more info about that

Reply:
The wrong order of checks:
1) first we checked that user with the same already exists
2) then we checked entered fields
We should make (2) then (1)
![43bu8rP7Sj](https://user-images.githubusercontent.com/12034551/95962883-a0341c00-0e0f-11eb-86e0-60b5b963a9ad.gif)

Reply:
all comments are addressed